java/src/test/java/ai/rapids/cudf/TableTest.java

```diff
@@ -6338,6 +6338,56 @@ void testBoundsCheckedGather() {
     }
   }
 
+
+  @Test
+  void testScatterTable() {
+    try (Table srcTable = new Table.TestBuilder()
+            .column(1, 2, 3, 4, 5)
+            .column("A", "AA", "AAA", "AAAA", "AAAAA")
+            .decimal32Column(-3, 1, 2, 3, 4, 5)
+            .decimal64Column(-8, 100001L, 200002L, 300003L, 400004L, 500005L)
```

Comment:
There are 6 rows here but the other columns in the table only have 5.  The target and expected tables have the same issues.

Reply:
No, these decimal columns only have 5 rows. The first number is scale. Yeah, such constructor is confusing.